src/makers.rs

```diff
@@ -21,6 +21,8 @@ fn get_name() -> String {
 }
 
 fn main() {
+    #[cfg(windows)]
+    let _ = ansi_term::enable_ansi_support();
```

Comment:
question, do we need the let _ here?

Reply:
Alternative would be something like:
```
#[allow(unused_must_use)]
{
      #[cfg(windows)]
      ansi_term::enable_ansi_support();
}
```
I didn't test it but I've found this workaround on stackoverflow or a Rust forum / issue. 

This doesn't work for some reasons:
```rust
#[allow(unused_must_use)]
#[cfg(windows)]
ansi_term::enable_ansi_support();
```

Reply:
lets leave it. thanks for the explanation.